ibl_to_nwb/metadata/ecephys.yml

```
@@ -82,3 +82,7 @@ Ecephys:
       description: Proportion of quality metrics passed.
     - name: allen_location
       description: Brain region reference in the Allen Mouse Brain Atlas.
+    - name: uuid
```

```suggestion
    - name: global_id
```

It is best practice for coregistration across sessions on DANDI to refer to these IDs in a more immediately obvious way

(the column name in the extractor should also be updated to reflect this change)

---

Since the recordings are acute, the uuid refers to a single cluster in a single recording, and can not be used to link across sessions. The only real use I can see is to compare / verify a clusters identity between the .nwb and ONE derived easily, I would suggest to leave the name of the variable identical in both.

---

I see, thanks for clarifying